### DISCUSSION TITLE
What is the difference between the two?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/devtools-v6#1677] [Q&A]
![image](https://user-images.githubusercontent.com/31676163/152774556-bcd6008e-c821-4f88-ae50-2ef49edb4ece.png)

**Comment by Akryum** [comment]
Left: beta channel | Right: stable channel

You shouldn't have multiple versions enabled at the same time BTW.

**Reply by JenuelDev** [reply]
is the 5.3.4  supports vue 3?

**Reply by Akryum** [reply]
No

**Reply by JenuelDev** [reply]
ah,, that means we have to use the beta which is unstable? or is their any alternative?

**Reply by Akryum** [reply]
Google is still validating v6.0.0 on the stable channel. If you can't wait I suggest using Firefox.
